client/src/Components/Layouts/NotFound/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.css';
+
+export default function index() {
+    return (
+      <section className="pageNotFoundMain">
+        <div>
```

Comment:
@denaHS You might be able to remove this div without changing the appearance of the page.